### ISSUE TITLE
Feature Request: Show the max number of students in the course if possible

### ISSUE THREAD

**Issue by Shimushushushu** [SJTU-Plus/course-plus#80]
Just as the title said, this is pretty useful if students want to check out whether they could join a course before the second turn of joining courses begins, if there's a way to getting know of these info (although it seems that we can not check this out at `i.sjtu.edu.cn/design/funcData_cxFuncDataList.html`) .

Also, it would be better if currently how many students have joined the course are shown as well.

**Comment by skyzh** [comment]
Thanks for your advice!

By the way, our data fetcher is at https://github.com/SJTU-Plus/course-plus-data/tree/master/course_plus_data_fetcher . If possible, you could also help us implement this functionality.

**Comment by Shimushushushu** [comment]
Hmm, put improving the fetcher aside (I'm afraid I'm not capable), actually I think they have already been fetched and stored in those JSON files, whose names are "jxbrs" (教学班人数) and "xkrs" (选课人数) respectively.

What has hit my mind is that adding this feature means that data may have to be updated a bit more frequently, at least after each turn of joining courses.

**Comment by skyzh** [comment]
Yes, I just found these data has already been fetched. We could later show them on frontend and update them after each round. This is some kind of future work, and may not be implemented in this semester.

**Comment by Shimushushushu** [comment]
Yes and I fully understand that.  )-:

So I'll leave the issue alone and probably not comment on it until there's new progress.
